clients/src/main/java/org/apache/kafka/clients/consumer/internals/NetworkClientDelegate.java

```diff
@@ -267,7 +275,9 @@ public void addAll(final List<UnsentRequest> requests) {
     public void add(final UnsentRequest r) {
         Objects.requireNonNull(r);
         r.setTimer(this.time, this.requestTimeoutMs);
+        r.setEnqueuedMs(this.time.milliseconds());
```

Comment:
Is it OK to use `this.time.milliseconds()` here? Either way, I think we can call `time.milliseconds()` once and reuse it in the metrics update too, right?

Reply:
After discussing with @lianetm, we may just want to record unsent request size at the end of `poll`, so I remove the metrics update here.

https://github.com/apache/kafka/pull/17199#discussion_r1849097694